src/test/java/org/xstefank/check/TestUtilities.java

```diff
@@ -0,0 +1,64 @@
+package org.xstefank.check;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.BeforeClass;
+import org.powermock.api.mockito.PowerMockito;
+import org.xstefank.api.GitHubAPI;
+import org.xstefank.model.yaml.Format;
+import org.xstefank.model.yaml.FormatConfig;
+import org.xstefank.model.yaml.SkipPatterns;
+import java.io.File;
+import java.io.IOException;
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+import static org.powermock.api.support.membermodification.MemberModifier.suppress;
+
+public class TestUtilities {
+
+    private static final String SUPPRESS_METHOD_NAME = "readToken";
+    private static final String STUB_METHOD_NAME = "getJsonWithCommits";
+    private static final String TEST_PULL_REQUEST_PAYLOAD = "testPullRequestPayload.json";
+    private static final String TEST_COMMITS_PAYLOAD = "testCommitsPayload.json";
+    private static final String TEST_SIMPLE_PULL_REQUEST_PAYLOAD = "testSimplePullRequestPayload.json";
+
+    protected static JsonNode testPullRequestPayload;
+    protected static JsonNode testCommitsPayload;
+    protected static JsonNode testSimplePullRequestPayload;
+
+    @BeforeClass
+    public static void beforeClass() throws IOException {
+        testPullRequestPayload = TestUtilities.getTestPullRequestPayload();
+        testCommitsPayload = TestUtilities.getTestCommitsPayload();
+        testSimplePullRequestPayload = TestUtilities.getTestSimplePullRequestPayload();
+    }
+
+    public static JsonNode readJsonNode(String resource) throws IOException {
+        File testFile = new File(SkipCheck.class.getClassLoader().getResource(resource).getFile());
+        return new ObjectMapper().readTree(testFile);
+    }
+
+    public static FormatConfig setUpFormatConfig(SkipPatterns testSkipPatterns) {
+        Format testFormat = new Format();
+        testFormat.setSkipPatterns(testSkipPatterns);
+        FormatConfig testFormatConfig = new FormatConfig();
+        testFormatConfig.setFormat(testFormat);
+        return testFormatConfig;
+    }
+
+    public static void stubMethodWithPowerMock(JsonNode testCommitsPayload) {
+        suppress(method(GitHubAPI.class, SUPPRESS_METHOD_NAME));
```

Comment:
you can name this constant with reference of the name of the suppressed method, in that case, when we will need to suppress another method there is no need to rename also this one